CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -114,7 +110,7 @@ local EaseInOutQuad = function(t, b, c, d)
 	if t >= d then return b + c end
 
 	t = t / (d/2);
-	if (t < 1) then return c/2*t*t + b end;
+	if t < 1 then return c/2*t*t + b end;
 	t = t - 1;
 	return -c/2 * (t*(t-2) - 1) + b;
```

Comment:
And remove this semicolon, should just make it all consistent. Single line if-statements and semicolons have no place in Lua ;)